mentoring/mentor-guide.md

```diff
@@ -0,0 +1,100 @@
+*  picking issues that don’t span multiple SIGs for new contributors
+*  or they do because they are learning to be a better approver
+*  labeling that will help them identify clear issues that need their attention
+* Make introductions to resources and people for the mentee - this is sponsorship which is now equally as important as mentorship.
+* Help contributor be productive.
+* Patience and empathy.  Through repetition, people learn, and as a mentor, we cannot expect a mentee to meet all of our expectations.  Through the removal of expectation and bias, and with a sprinkle of compassion, we can gain more patience.
+* Have some fun and learn together!
+
+
+### Responsibilities by each program
+*[Group mentoring](group-mentoring.md)*
+
+You’ll need to be:
+*  On Slack at least 1-2 hours a week fielding questions.
+*  Available for a group bi-weekly standup to check the progress of the mentees. 
+*  Helpful with building comradery amongst the group - this can be as simple as curating conversation in slack.
+
+Mentees will be asked to take responsibility for their growth and improvement as it's a semi-structured self-paced program; however, you are there to help guide them by sharing your methodologies, resources, tips, and experiences.
+
+*[The 1:1 Hour](the1-on-1.md)*
+* Participate in a one hour a month session via zoom video conference; no more than 3 hours/times and that is opt-in.
+* Depending on the activities that you’ve selected as comfortable in helping with, the mentee may ask:
+*  To pair program with you.
+*  Ask you an hours worth of questions (AMA). 
+*  Do code reviews/docs reviews.
+*  Discuss their path so far and ask for input/feedback.
+*  Code base tour of a certain area
+*  Or a mix - it’s up to the mentee with how they’d like to spend the time and your skills/experience.
+* You'll fill out a form once a month to list your availibility.
+
+*[Outreachy](https://www.outreachy.org)*
+* Being a mentor can take anywhere from 2-5 hours a week depending on the scope of the project that the intern(s) work on. 
+* Before the program issue grooming/project planning activities:
+  * For the Outreachy application process that they host [here](https://www.outreachy.org/apply/). All interns that are interested in the project must make one contribution as part of their application. Label them with a good first time issue.
+* During the program:
+  * The time above is split up between slack, 1:1 pair programming and standups.
+  * Provide official feedback during intervals to Outreachy and the intern.
+  * Have read and understand the [rules and guidelines](https://www.outreachy.org/mentor/) around mentoring for Outreachy
+
+CNCF will financially support one Outreachy intern for Kubernetes; if you would like to participate and we already have one, assistance with securing a corporate sponsor for the budget (6.5k) may be needed.
```

Comment:
This is reading a little odd now to me.  Suggest: "...for the additional budget..."